app/package/models.py

```diff
@@ -335,7 +340,8 @@ class MetaDataDB(db.Model):
     name = db.Column(db.TEXT, index=True)
     version = db.Column(db.TEXT, index=True, default='latest')
     descriptor = db.Column(db.JSON)
-    status = db.Column(db.TEXT, index=True, default='active')
+    status = db.Column(db.Enum(PackageStateEnum, native_enum=False),
+                       index=True, default=PackageStateEnum.active)
```

Comment:
@Fak3 First of all. I know how B-Tree works. 
And now lets comes to the answer
- Case 1: We do not give index to. What will happen?
The optimizer really does not have any option other than scan table.
- Case 2: We do create index. What will happen?
We are giving optimizer options to choose between 1. scan or 2. index lookup. 
Now based on your data the query optimizer will decide what to use. 

Now comes to the problem with write latency. Our system is supposed to be more read heavier than write. So i can give away some more milli second for write if i can save some milli seconds in read.

```The myth of low cardinality indexes not being useful or used... seriously needs to be discarded.```